.gitpod/setup-env.sh

```diff
@@ -0,0 +1,12 @@
+# Initialize variables to get Moodle code.
+if [ -z "$MOODLE_REPOSITORY" ];
+then
+    export MOODLE_REPOSITORY=https://github.com/moodle/moodle.git
```

Comment:
Q: Sorry, missed this in the review. For your consideration 100%. Should we point to the canonical server (`https://git.in.moodle.com/moodle/moodle`) or github is, in practice, the one most used out there.

Reply:
I'm happy with any of the 2 options. IMHO, GitHub is the most used (that's why I thought it was better to use it as default) so, for now, I left GitHub URL (although I can easily change it if required) :-)

Reply:
Nah, I'm ok (and less load to our one :-) )